manifests/plugin/powerdns.pp

```diff
@@ -3,13 +3,16 @@
   Enum['present', 'absent'] $ensure       = 'present',
   Integer $order                          = 10,
   Optional[Numeric] $interval             = undef,
-  Optional[Hash[String, Hash]] $servers   = undef,
-  Optional[Hash[String, Hash]] $recursors = undef,
+  Optional[Hash[String, Hash]] $servers   = {},
```

Comment:
Can we remove the `Optional`?

Reply:
yeah sure,  i didn't want to change too much! just doing the bare minimum to get create_resources replaced

Reply:
@alexjfisher  removed